config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/CertificatesLoader.java

```diff
@@ -74,6 +74,11 @@ public Object loadSingle(String name, File entitiesFile) {
         return fileUtils.getFileAsString(entitiesFile);
     }
 
+    @Override
+    public Map<String, Object> load(File entitiesFile) {
+        throw new ConfigLoadException("Cannot load certificates from config file");
```

Comment:
@uppoju what is our logic here, overriding the load and throwing cannot load exception

Reply:
we added this new method in EntityLoader interface for loading environment entities, So this method does not support loading all certificate files from config folder
It should be done from trusted-cert.yml file

Reply:
OK, then we can return null instead throwing exception

Reply:
if method is not supported we should throw exception saying this is not supported, we are doing the same in existing unsupported methods ex. loadSingle in PolicyAndFolderLoader.java